crypto/fipsmodule/evp/digestsign.c

```diff
@@ -130,7 +144,7 @@ int EVP_DigestVerifyInit(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
 }
 
```

Comment:
Need to bail early for these functions if called with an HMAC type pkey?
* EVP_DigestVerifyFinal
* EVP_DigestVerify

Reply:
Added https://github.com/aws/aws-lc/pull/1324/commits/984fccc3f6bd9416be8f3324af1a02627a0a6ff3 per our discussion :)